eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -19,7 +19,11 @@ param (
   [Parameter(Mandatory = $true)]
   $Language, # EG: js, java, dotnet. Used in language for the embedded readme.
   [Parameter(Mandatory = $true)]
-  $Configs # The configuration elements informing important locations within the cloned doc repo
+  $Configs, # The configuration elements informing important locations within the cloned doc repo
+
+  # arguments for metadata author information
+  [Parameter(Mandatory = $false)]
+  $CodeOwners # Code owners github identity.
```

Comment:
```suggestion
  [string[]] $CodeOwners # Code owners github identity.
```
We should declare this as an array since you are using it as such below.